### PR TITLE
SG-22646-shotgrid Shotgun instances has been replaced

### PR DESCRIPTION
**Pull request by zamu5**

### CODE REVIEW

File: docs/search_completer.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Shotgun Search Completers
+Search Completers
```
Comment (jfboismenu) [inline docs/search_completer.rst:1]:
```suggestion
ShotGrid Search Completers
```

File: docs/search_completer.rst [hunk 2]
```diff
@@ -9,7 +9,7 @@ Global Search Completer
 =======================
 
 The global search completer is a QCompleter instance for use with widgets
-that allow input of Shotgun entities. The completer can be customized to
+that allow input of  entities. The completer can be customized to
```
Comment (jfboismenu) [inline docs/search_completer.rst:12]:
```suggestion
that allow input of ShotGrid entities. The completer can be customized to
```

File: python/shotgun_fields/shotgun_field_manager.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def get_class(cls, sg_entity_type, field_name, widget_type=DISPLAY):
         """
         if widget_type not in cls._WIDGET_TYPES:
             raise ValueError(
-                "ShotgunFieldManager unable to retrieve fields of type: %s "
+                "ShotGridFieldManager unable to retrieve fields of type: %s "
```
Comment (jfboismenu) [inline python/shotgun_fields/shotgun_field_manager.py:75]:
```suggestion
                "ShotgunFieldManager unable to retrieve fields of type: %s "
```

File: python/shotgun_fields/shotgun_field_manager.py [hunk 4]
```diff
@@ -119,7 +119,7 @@ def register_class(cls, field_type, widget_class, widget_type):
 
         if widget_type not in cls._WIDGET_TYPES:
             raise ValueError(
-                "ShotgunFieldManager unable to register unrecognized widget type: %s "
+                "ShotGridFieldManager unable to register unrecognized widget type: %s "
```
Comment (jfboismenu) [inline python/shotgun_fields/shotgun_field_manager.py:122]:
```suggestion
                "ShotgunFieldManager unable to register unrecognized widget type: %s "
```

File: python/shotgun_fields/shotgun_field_manager.py [hunk 5]
```diff
@@ -148,7 +148,7 @@ def register_entity_field_class(
 
         if widget_type not in cls._WIDGET_TYPES:
             raise ValueError(
-                "ShotgunFieldManager unable to register unrecognized widgets type: %s "
+                "ShotGridFieldManager unable to register unrecognized widgets type: %s "
```
Comment (jfboismenu) [inline python/shotgun_fields/shotgun_field_manager.py:151]:
```suggestion
                "ShotgunFieldManager unable to register unrecognized widgets type: %s "
```

File: python/shotgun_fields/shotgun_field_manager.py [hunk 6]
```diff
@@ -331,7 +331,7 @@ def create_widget(
             )
         else:
             raise TypeError(
-                "Unknown widget type supplied to ShotgunFieldManager."
+                "Unknown widget type supplied to ShotGridFieldManager."
```
Comment (jfboismenu) [inline python/shotgun_fields/shotgun_field_manager.py:334]:
```suggestion
                "Unknown widget type supplied to ShotgunFieldManager."
```

File: python/shotgun_fields/shotgun_field_meta.py [hunk 7]
```diff
@@ -150,11 +150,11 @@ def __new__(mcl, name, parents, class_dict):
         # validate the class definition to make sure it implements the needed interface
         if ("_DISPLAY_TYPE" not in class_dict) and ("_EDITOR_TYPE" not in class_dict):
             raise ValueError(
-                "ShotgunFieldMeta classes must have a _DISPLAY_TYPE or _EDITOR_TYPE member variable"
+                "ShotGridFieldMeta classes must have a _DISPLAY_TYPE or _EDITOR_TYPE member variable"
```
Comment (jfboismenu) [inline python/shotgun_fields/shotgun_field_meta.py:153]:
```suggestion
                "ShotgunFieldMeta classes must have a _DISPLAY_TYPE or _EDITOR_TYPE member variable"
```

File: python/shotgun_fields/shotgun_field_meta.py [hunk 8]
```diff
@@ -150,11 +150,11 @@ def __new__(mcl, name, parents, class_dict):
...
             )
         if "__init__" in class_dict:
             raise ValueError(
-                "ShotgunFieldMeta classes cannot define their own constructor"
+                "ShotGridFieldMeta classes cannot define their own constructor"
```
Comment (jfboismenu) [inline python/shotgun_fields/shotgun_field_meta.py:157]:
```suggestion
                "ShotgunFieldMeta classes cannot define their own constructor"
```

File: python/shotgun_fields/widget_metaclass.py [hunk 9]
```diff
@@ -75,11 +75,11 @@ def __new__(mcl, name, parents, class_dict):
         # validate the class definition to make sure it implements the needed interface
         if "_FIELD_TYPE" not in class_dict:
             raise ValueError(
-                "ShotgunFieldMeta classes must have a _FIELD_TYPE member variable"
+                "ShotGridFieldMeta classes must have a _FIELD_TYPE member variable"
```
Comment (jfboismenu) [inline python/shotgun_fields/widget_metaclass.py:78]:
```suggestion
                "ShotgunFieldMeta classes must have a _FIELD_TYPE member variable"
```

File: python/shotgun_fields/widget_metaclass.py [hunk 10]
```diff
@@ -75,11 +75,11 @@ def __new__(mcl, name, parents, class_dict):
...
             )
         if "__init__" in class_dict:
             raise ValueError(
-                "ShotgunFieldMeta classes cannot define their own constructor"
+                "ShotGridFieldMeta classes cannot define their own constructor"
```
Comment (jfboismenu) [inline python/shotgun_fields/widget_metaclass.py:82]:
```suggestion
                "ShotgunFieldMeta classes cannot define their own constructor"
```

File: python/version_details/shotgun_entities/card_widget.py [hunk 11]
```diff
@@ -68,7 +68,7 @@ def add_field(self, field_name, label_exempt=False):
         """
         if not self.field_manager:
             raise RuntimeError(
-                "No ShotgunFieldManager has been set, unable to add fields."
+                "No ShotGridFieldManager has been set, unable to add fields."
```
Comment (jfboismenu) [inline python/version_details/shotgun_entities/card_widget.py:71]:
```suggestion
                "No ShotgunFieldManager has been set, unable to add fields."
```

File: python/version_details/shotgun_entities/card_widget.py [hunk 12]
```diff
@@ -316,7 +316,7 @@ def _get_entity(self):
     def _set_entity(self, entity):
         if not self.field_manager:
             raise RuntimeError(
-                "No ShotgunFieldManager has been set, unable to set entity."
+                "No ShotGridFieldManager has been set, unable to set entity."
```
Comment (jfboismenu) [inline python/version_details/shotgun_entities/card_widget.py:319]:
```suggestion
                "No ShotgunFieldManager has been set, unable to set entity."
```

**Review by jfboismenu — CHANGES_REQUESTED** [review]
Looks good to be, but a few of the changes refer to class names, so I've suggested changes to make it easier for you to fix.

**Review by jfboismenu — APPROVED** [review]

**Review by mathurf — APPROVED** [review]
LGTM
Tested with Shotgunpanel and Demo app

**Pull request merged**